src/expressRequest.d.ts

```diff
@@ -0,0 +1,6 @@
+declare namespace Express {
+    export interface Request {
```

Comment:
Why?

Reply:
I think that it may be a bit misleading. 
Maybe put it a bit deeper. Not every request needs `id` and yet you declare this type globally.